src/client/common/configSettings.ts

```diff
@@ -92,6 +92,8 @@ export class JupyterSettings implements IWatchableJupyterSettings {
     public interactiveWindowMode: InteractiveWindowMode = 'multiple';
     // Hidden settings not surfaced in package.json
     public disableZMQSupport: boolean = false;
+    // Hidden settings not surfaced in package.json
+    public disablePythonDaemon: boolean = false;
```

Comment:
This applies to unix systems.
If daemon doesn't work, then python kernel cannot be interrupted (just like any other non-python kernel)